ompi/mca/coll/libnbc/libdict/dict_private.h

```diff
@@ -3,6 +3,7 @@
  *
  * Private definitions for libdict.
  * Copyright (C) 2001 Farooq Mela.
+ * Copyright (c) 2022 IBM Corporation.  All rights reserved.
```

Comment:
Trivial: this looks like an errant copyright change.

Reply:
I had changed this file, backed out changes, now copyright is removed.